library/amazonlinux

```diff
@@ -4,11 +4,19 @@ Maintainers: Amazon Linux Team <amazon-linux-ami@amazon.com> (@aws),
              Eric Warehime (@Eric-Warehime)
 GitRepo: https://github.com/aws/amazon-linux-docker-images.git
 
-Tags: 2017.03.0.20170812, 2017.03, latest
+Tags: 2017.09.0.20170930, 2017.09, latest
+GitFetch: refs/heads/2017.09
+GitCommit: eb819d6b91ada2f0304d2a24ab9ce2ced0f166ea
+
+Tags: 2017.09.0.20170930-with-sources, 2017.09-with-sources, with-sources
+GitFetch: refs/heads/2017.09-with-sources
+GitCommit: f13ce885778e5cf48275986b7cf8c321a60b6693
+
+Tags: 2017.03.1.20170812, 2017.03, latest
```

Comment:
You can't have two `latest` tags. :wink: (and `with-sources`).

Reply:
oh duh, hold on